src/bitbucket/descriptor.ts

```diff
@@ -1,80 +1,93 @@
 import { config } from '../lib/Config';
 
+const getAppName = () =>
+  `${config.name || 'Landkid'} ${(process.env.LANDKID_DEPLOYMENT || 'local').toUpperCase()}`;
+
 export const getAppKey = () =>
   `${config.key || 'landkid'}-${process.env.LANDKID_DEPLOYMENT || 'local'}`;
 
-export const makeDescriptor = () => ({
-  name: `${config.name || 'Landkid'} ${(process.env.LANDKID_DEPLOYMENT || 'local').toUpperCase()}`,
-  description: "Addon to display a 'release queue' panel in PRs",
-  baseUrl: config.baseUrl,
-  key: getAppKey(),
-  vendor: {
-    name: 'Fabric Build',
-  },
-  // :write implies base scope (e.g. pullrequest:write implies pullrequest)
-  // pullrequest:write requires repository:write so we don't need to define that as well
-  scopes: ['account', 'pullrequest:write', 'pipeline:write'],
-  authentication: {
-    type: 'jwt',
-  },
-  contexts: ['account'],
-  lifecycle: {
-    installed: '/bitbucket/lifecycle/installed',
-    uninstalled: '/bitbucket/lifecycle/uninstalled',
-  },
-  modules: {
-    proxy: {
-      '/can-land/{repository}/{pullrequest}': {
-        destination: '/bitbucket/proxy/can-land?aaid={user.uuid}&pullRequestId={pullrequest.id}',
-      },
-      '/land/{repository}/{pullrequest}': {
-        destination:
-          '/bitbucket/proxy/land?aaid={user.uuid}&pullRequestId={pullrequest.id}&commit={pullrequest.source.commit.hash}',
-      },
-      '/land-when-able/{repository}/{pullrequest}': {
-        destination:
-          '/bitbucket/proxy/land-when-able?aaid={user.uuid}&pullRequestId={pullrequest.id}&commit={pullrequest.source.commit.hash}',
-      },
+export const makeDescriptor = () => {
+  const appName = getAppName();
+  const appKey = getAppKey();
+
+  const addonName = `Landkid Queue${
+    process.env.LANDKID_DEPLOYMENT !== 'prod'
+      ? ` (${process.env.LANDKID_DEPLOYMENT || 'local'})`
+      : ''
+  }`;
+
+  const params = new URLSearchParams();
+  params.append('appName', appName);
+  const appNameQueryString = params.toString();
```

Comment:
Passing the app name in as a query string param was the only way I could figure out to access it from the addon.

Reply:
Hacky solution, but could we bake it into `src/static/bitbucket/components/App.tsx` directly? Would need to do something like adding it as an env var at build time though. Which is super doable, but is it more or less annoying that getting the addon refreshed? 🤔